RimWorld/RimWorld/Tool.cpp

```diff
@@ -0,0 +1,111 @@
+#include "Tool.h"
+
+Tool::Tool() : m_isPlacing(false), m_isZoning(false)
```

Comment:
I wonder why you define here, what are the initial values for these since you have those outside in your main. Should be linked to values in main then for consistency. For example, forcing them to be passed in c-tor